18F/guides
# Update de-risking guide PDF to 9/27/2024 version

## Changes proposed in this pull request:

This updates the downloadable PDF version of the de-risking guide to acknowledge the 10x program in the document history.

Resolves https://github.com/18F/guides/issues/709.

## Testing

- Visit a de-risking guide in this [PR build](https://federalist-8bc0b42c-5fd0-4ae8-9366-cd9c265a4446.sites.pages.cloud.gov/preview/18f/guides/guides-709-update-pdf/derisking-government-tech/)
- Confirm that clicking Download in the primary nav gives you the latest version of the PDF

## security considerations

None. This replaces a downloadable file.
